content/guides/forecasts.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Forecasts
+kind: guide
+listorder: 24
+beta: true
+---
+
+Forecasting is an algorithmic feature that allows you to predict where
+a metric is heading in the future. It is well-suited for metrics with
+strong trends or recurring patterns.
+
+For example, if your application starts logging at a faster rate, forecasts can alert you a week before your disk fills up, giving you adequate time to update your log rotation policy. Or, you can forecast business metrics, such as user sign-ups, to track progress against your quarterly targets.
+
+## How to Forecast Your Data
+
+There is a `forecast` function in the Datadog query language. When you apply this function to a series, it returns the usual results along with a forecast of future values.
+
+### Visualizing Forecasts in Dashboards
+
+The chart below shows a dashboard tile of a forecast in “live” mode. The dashed pink line divides the history from the forecast, and the current time is indicated by the gray “Now” marker and line. The dotted green line shows the future values predicted by the forecast, and the range of predicted values within 3 deviations (a tunable parameter) is shown in lighter green.
+
+{{< img src="forecasts/dashboard_live.png" >}}
+
+When the dashboard is no longer in “live” mode, the vertical pink line and the gray line indicating “Now” will start to diverge. In the chart below, the dashed pink line is still splitting the history from the forecast, but we can see actual data (solid green) to the right of the pink line. The lighter green is still showing the range of predicted values and the dotted green line is still showing the predicted future values. The forecast is based only on the actual data observed before the dashed pink line. This view is useful for checking how accurate the forecast is against known data.
+
+{{< img src="forecasts/dashboard_past.png" >}}
+
+To create a forecast graph, start by adding a timeseries graph to your dashboard. Be sure to select “Timeseries” as the visualization type. Now, click on the **+** icon on the right side of your expression. Choose the **"Forecast”** function in the **“Algorithms”** submenu. You should immediately see the preview update to include the forecast visualization. A number of the graphing options will disappear, as forecasts have a unique visualization.
+
+The function has two parameters. The first parameter is for selecting which algorithm (see [Forecast Algorithms](guides/forecasts/#forecast-algorithms)) will be used. The second parameter is deviations, and you can tune this to change the width of the range of forecasted values. A value of 1 or 2 should be large enough to accurately forecast most “normal” points. After successfully adding forecast, your editor should show something like this:
```

Comment:
Nit: `forecast` in quotes, or `the forecast function`.

Reply:
I think the link should just be `#forecast-algorithms` (it doesn't work in the staging preview).